### PR TITLE
Pytorch cifar data set tutorial

### PR DESCRIPTION
**Pull request by Tadeoz**
I was tuning with parameter changes to transformation, batch sizes, and epochs—the optim.Adam function was altered in previous model runs. I was using Optim.Adam(net.parameters(), lr = .002). The accuracy increased but the validation accuracy graph was extremely inconsistent.

### CODE REVIEW

**Review by michaelmunje — COMMENTED** [review]
File does not load but will merge in for now. We will delete all these files in a cleanup later.

**Review by michaelmunje — APPROVED** [review]
